build-system/tasks/e2e/build-bento-e2e.js

```diff
@@ -0,0 +1,133 @@
+const {compileCss} = require('../css');
+const {
+  buildBinaries,
+  buildExtensions,
+  buildNpmCss,
+} = require('../extension-helpers');
+
+const fastGlob = require('fast-glob');
+const {endBuildStep} = require('../helpers');
+const path = require('path');
+const {existsSync, mkdirSync, writeFile} = require('fs-extra');
+
+/**
+ * Build the Bento end to end page
+ *
+ * @param {!Object} options
+ * @return {!Promise<void>}
+ */
+async function buildBentoE2E(options) {
+  const e2eFixtures = await fastGlob(
+    'extensions/*/1.0/test-e2e/e2e-fixture.js'
+  );
+
+  const isMissingCss =
+    (await fastGlob(`src/bento/components/*/1.0/dist/styles.css`)).length === 0;
+  if (isMissingCss) {
+    await compileCss(options);
+    await buildExtensions({...options, compileOnlyCss: true});
+  }
+
+  const results = e2eFixtures.flatMap((fixturePath) => {
+    const newOptions = {
+      ...options,
+      // outputting builds to test directory
+      outputPath: 'test/fixtures/e2e/',
+    };
+    const bentoComponentName = fixturePath.match(/amp-(.*(?=\/\d))/)?.[1];
```

Comment:
Checkout the `getBentoName` function
https://github.com/ampproject/amphtml/blob/a6546523dace584b5bb353e004df8b5481a8c051/build-system/tasks/bento-helpers.js#L5

Reply:
I'm not sure if I can use this?

Maybe I shouldn't use fastGlob to find the files that need to be build. But since I have a path, I need to find the component name in there, and there is still at least one time I need to keep the `amp-component-name`